deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -2688,6 +2688,20 @@ fun(Conf) ->
     end
 end}.
 
+{mapping, "cluster_tags.$tag", "rabbit.cluster_tags", [
+    {datatype, [string]}
```

Comment:
You can change this to `binary` as of https://github.com/Kyorai/cuttlefish/commit/d367bb702471f9e13b1a06f5a5d47a3e5d766eb7 (already used by RabbitMQ) and avoid a conversion to binary in other places.

Reply:
good point, will do.